test/utils/Deployers.sol

```diff
@@ -87,15 +87,6 @@ contract Deployers {
     uint160 hookPermissionCount = 14;
     uint160 clearAllHookPermissionsMask = ~uint160(0) << (hookPermissionCount);
 
-    modifier noIsolate() {
```

Comment:
What was this from?

Reply:
it was so the tests could call `sync` in a separate call than the actions router stuff. But now sync has to be in there to be in the lock so we dont need it